crates/biome_fs/src/fs/memory.rs

```diff
@@ -166,6 +183,16 @@ impl FileSystem for MemoryFileSystem {
         let files = self.files.0.read();
         files.get(path).is_some()
     }
+
+    fn get_changed_files(&self, _base: &str) -> io::Result<Vec<String>> {
+        let cb_arc = self.on_get_changed_files.as_ref().unwrap().clone();
+
+        let mut cb_guard = cb_arc.lock();
+
+        let cb = cb_guard.take().unwrap();
```

Comment:
We should avoid `unwrap` as much as possible, unless we are 100% that is safe to call it. If we need to use `unwrap`, we usually use `expect` instead. `expect` accepts string that we can use to send a hint.